components/card/card-loading.component.ts

```diff
@@ -11,15 +11,15 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
   exportAs: 'nzCardLoading',
   template: `
     <div class="ant-card-loading-content">
-      <div class="ant-row" style="margin-left: -4px; margin-right: -4px;" *ngFor="let listOfClassName of listOfLoading">
-        <div
-          *ngFor="let className of listOfClassName"
-          [ngClass]="className"
-          style="padding-left: 4px; padding-right: 4px;"
-        >
-          <div class="ant-card-loading-block"></div>
+      @for (listOfClassName of listOfLoading; track $index) {
```

Comment:
`track： listOfClassName`

Reply:
> `track： listOfClassName`

i'm not sure that we will improve perfromance by tracking on an array but i can make the change

Reply:
In my opinion, using index as track is not reliable.  In contrast, using an array (object reference) here is more reliable and safe than index.

I hope we can agree on this :)